src/main/scala/com/github/dgilling/datetimeroutes/JodaDateTimeRoutes.scala

```diff
@@ -105,7 +106,9 @@ object JodaPeriodConverter {
         val time = DateTime.now(dateTimeZone).plusDays((sign + value).toInt)
         intervalResetBoundaryFcn.map(_(Daily, time)).getOrElse(time)
       case WeeksRegex(sign, value) =>
-        val time = DateTime.now(dateTimeZone).plusWeeks((sign + value).toInt)
+        val currentDayOfWeek = DateTime.now(dateTimeZone).dayOfWeek().get()
+        val delta = if (weekStartsOn > currentDayOfWeek) (value.toInt + 1).toString else value
```

Comment:
do we plan to send this only for workspaces that not funnel or retention graphs? for funnels and retention, its possible.. this may break labels etc. between what ui expects and backend creates.. need to test. otherwise lgtm

Reply:
Yea, plan for now is not retention and funnels.